tests/CacheTool/Command/ApcBinDumpCommandTest.php

```diff
@@ -6,6 +6,9 @@ class ApcBinDumpCommandTest extends CommandTest
 {
     public function testCommand()
     {
+        if (explode('.', PHP_VERSION_ID)[0] >= 7) {
```

Comment:
Also you can't explode on '.' because PHP_VERSION_ID has no dots, it's an integer MAJOR/MINOR/PATCH without separator. See http://php.net/manual/en/reserved.constants.php#reserved.constants.core